fastlane/Helpers/Version.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Version.swift
+//  FastlaneRunner
+//
+//  Created by Khanh on 26/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+enum Version {
+    
+    // MARK: - Getting
+
+    static func getVersionNumber() -> String {
```

Comment:
Let's use the variable instead
```swift
static var versionNumber: String {
	get {
		getVersionNumber(
			xcodeproj: .userDefined(Constant.projectPath),
			target: .userDefined(Constant.projectName)
		)
	}

	set {
		incrementVersionNumber(
            versionNumber: .userDefined(newValue),
            xcodeproj: .userDefined(Constant.projectPath)
        )
	}
}
```

Same with build number.

Reply:
This way is much cleaner, I updated at https://github.com/nimblehq/ios-templates/pull/362/commits/68a1a4b202f5d4618c00f973183cb2c433848d1a. Thank you 🙏 .